recipe/meta.yaml

```diff
@@ -15,7 +15,7 @@ build:
 
 requirements:
   host:
-    - python >=3.6
+    - python
```

Comment:
thanks, just add to the "build:" section the skip for python less 3.6

Reply:
the skip section was updated:

https://github.com/AnacondaRecipes/bleach-feedstock/blob/update_to_4.1.0/recipe/meta.yaml#L15